drivers/serial/CMakeLists.txt

```diff
@@ -64,6 +64,7 @@ zephyr_library_sources_ifdef(CONFIG_UART_NUMICRO uart_numicro.c)
 zephyr_library_sources_ifdef(CONFIG_UART_NXP_S32_LINFLEXD uart_nxp_s32_linflexd.c)
 zephyr_library_sources_ifdef(CONFIG_UART_OPENTITAN uart_opentitan.c)
 zephyr_library_sources_ifdef(CONFIG_UART_PDL_INFINEON_CAT1 uart_ifx_cat1_pdl.c)
+zephyr_library_sources_ifdef(CONFIG_UART_PIC32CXSG uart_sam0.c)
```

Comment:
This driver support only one binding at moment.
https://github.com/zephyrproject-rtos/zephyr/blob/aa6f87dcbfeeb21cc43981a185103e062deaa1c1/drivers/serial/uart_sam0.c#L8